### PR TITLE
Refine Debian sources

### PR DESCRIPTION
**Pull request by pombredanne**
@gernot-h for your review some minor refinements on top of your PR.

### CODE REVIEW

File: PURL-TYPES.rst [hunk 1]
```diff
@@ -130,12 +130,14 @@ deb
...
 - The ``version`` is the version of the binary (or source) package.
-- ``arch`` is the qualifiers key for a package architecture. The special
-  value ``arch=source`` shall identify a Debian source package (which
-  usually consists of a Debian Source control file (.dsc) and corresponding
-  upstream and Debian sources). In this case, ``name`` and ``version`` of the
-  source package shall be given (both may differ from binary package!), as
-  printed by ``dpkg-query -f '${source:Package} ${source:Version} -W <pkg>``.
+- ``arch`` is the qualifiers key for a package architecture. The special value
+  ``arch=source`` identifies a Debian source package that usually consists of a
+  Debian Source control file (.dsc) and corresponding   upstream and Debian
```
Comment (gernot-h) [inline PURL-TYPES.rst]: There seems to be a whitespace issue: "corresponding   upstream". Do we care about those?

**Comment by gernot-h** [comment]
Besides the small whitespace issue mentioned above which I'm unsure if we (read: you ;) ) care about, this looks great, thanks for improving!

Just let me know if you want to rework or if I shall merge as-is.

**Comment by pombredanne** [comment]
@gernot-h Thanks! whitespace fixed.

**Comment by gernot-h** [comment]
Merged. I will squash your fixup commit, hope that's ok.

**Pull request merged**